lib/schema/manifest.go

```diff
@@ -1208,6 +1249,9 @@ func ShouldSkipApp(manifest Manifest, app loc.Locator) bool {
 		if ext != nil && ext.Catalog != nil && ext.Catalog.Disabled {
 			return true
 		}
+	case defaults.StorageAppName:
+		// do not install storage-app if no storage providers are enabled
+		return !manifest.OpenEBSEnabled()
```

Comment:
Is this complicated to also filter out the application package as a dependency if not enabled?

Reply:
You mean so it's not packaged during tele build? Yeah, this is something that's on my list of improvements, I haven't looked at what it'd take to implement this yet.